sparql/sparql11/protocol/manifest.ttl

```diff
@@ -1,961 +1,1430 @@
+  mf:name "invoke update operation with a POST with media type that's not url-encoded or application/sparql-update" .
+
+<#query_content_type_ask> a mf:ProtocolTest;
+  dawg:approval dawg:Approved;
+  dawg:approvedBy <http://www.w3.org/2009/sparql/meeting/2012-11-20#resolution_3>;
+  mf:action [
+    a ht:Connection;
+    ht:connectionAuthority "www.example";
+    ht:requests ([
+        a ht:Request;
+        ht:absolutePath "/sparql";
+        ht:body [
+          a cnt:ContentAsText;
+          cnt:characterEncoding "UTF-8";
+          cnt:chars "query=ASK%20%7B%7D&default-graph-uri=http%3A%2F%2Fkasei.us%2F2009%2F09%2Fsparql%2Fdata%2Fdata0.rdf"
+        ];
+        ht:httpVersion "1.1";
+        ht:methodName "POST";
+        ht:resp [
+          a ht:Response;
+          ht:headers ([
+              a ht:RequestHeader;
+              ht:fieldName "Content-Type";
+              ht:fieldValue "application/sparql-results+xml, application/sparql-results+json";
+              ht:headerElements ([
+                  a ht:HeaderElement;
+                  ht:elementName "application/sparql-results+xml"
+                ] [
+                  a ht:HeaderElement;
+                  ht:elementName "application/sparql-results+json"
+                ])
+            ]);
+          ht:statusCodeValue "2XX",
+            "3XX"
+        ]
+      ])
+  ];
+  mf:name "ASK query appropriate content type (expect one of: XML, JSON)" .
+
+<#query_content_type_construct> a mf:ProtocolTest;
+  dawg:approval dawg:Approved;
+  dawg:approvedBy <http://www.w3.org/2009/sparql/meeting/2012-11-20#resolution_3>;
+  mf:action [
+    a ht:Connection;
+    ht:connectionAuthority "www.example";
+    ht:requests ([
+        a ht:Request;
+        ht:absolutePath "/sparql";
+        ht:body [
+          a cnt:ContentAsText;
+          cnt:characterEncoding "UTF-8";
+          cnt:chars "query=CONSTRUCT%20%7B%20%3Cs%3E%20%3Cp%3E%201%20%7D%20WHERE%20%7B%7D&default-graph-uri=http%3A%2F%2Fkasei.us%2F2009%2F09%2Fsparql%2Fdata%2Fdata0.rdf"
+        ];
+        ht:httpVersion "1.1";
+        ht:methodName "POST";
+        ht:resp [
+          a ht:Response;
+          ht:headers ([
+              a ht:RequestHeader;
+              ht:fieldName "Content-Type";
+              ht:fieldValue "application/rdf+xml, text/turtle, application/n-triples, text/html, application/ld+json";
+              ht:headerElements ([
+                  a ht:HeaderElement;
+                  ht:elementName "application/rdf+xml"
+                ] [
+                  a ht:HeaderElement;
+                  ht:elementName "text/turtle"
+                ] [
+                  a ht:HeaderElement;
+                  ht:elementName "application/n-triples"
+                ] [
+                  a ht:HeaderElement;
+                  ht:elementName "text/html"
+                ] [
+                  a ht:HeaderElement;
+                  ht:elementName "application/ld+json"
+                ])
+            ]);
```

Comment:
Original test expected application/rdf+xml, application/rdf+json or text/turtle. This was probably made outdated by the RDF 1.1 WG, but want to call out that this is not the expectation of the original test which was approved by WG.